packages/babel-plugin-polyfill-corejs3/test/fixtures/entry-global/actual-corejs-3.20/output.mjs

```diff
@@ -0,0 +1,255 @@
+import "core-js/modules/es.string.split.js";
+import "core-js/modules/es.string.starts-with.js";
+import "core-js/modules/es.string.substr.js";
+import "core-js/modules/es.string.trim.js";
+import "core-js/modules/es.string.trim-end.js";
+import "core-js/modules/es.string.trim-start.js";
+import "core-js/modules/es.string.anchor.js";
+import "core-js/modules/es.string.big.js";
+import "core-js/modules/es.string.blink.js";
+import "core-js/modules/es.string.bold.js";
+import "core-js/modules/es.string.fixed.js";
+import "core-js/modules/es.string.fontcolor.js";
+import "core-js/modules/es.string.fontsize.js";
+import "core-js/modules/es.string.italics.js";
+import "core-js/modules/es.string.link.js";
+import "core-js/modules/es.string.small.js";
+import "core-js/modules/es.string.strike.js";
+import "core-js/modules/es.string.sub.js";
+import "core-js/modules/es.string.sup.js";
+import "core-js/modules/es.typed-array.float32-array.js";
+import "core-js/modules/es.typed-array.float64-array.js";
+import "core-js/modules/es.typed-array.int8-array.js";
+import "core-js/modules/es.typed-array.int16-array.js";
+import "core-js/modules/es.typed-array.int32-array.js";
+import "core-js/modules/es.typed-array.uint8-array.js";
+import "core-js/modules/es.typed-array.uint8-clamped-array.js";
+import "core-js/modules/es.typed-array.uint16-array.js";
+import "core-js/modules/es.typed-array.uint32-array.js";
+import "core-js/modules/es.typed-array.at.js";
+import "core-js/modules/es.typed-array.copy-within.js";
+import "core-js/modules/es.typed-array.every.js";
+import "core-js/modules/es.typed-array.fill.js";
+import "core-js/modules/es.typed-array.filter.js";
+import "core-js/modules/es.typed-array.find.js";
+import "core-js/modules/es.typed-array.find-index.js";
+import "core-js/modules/es.typed-array.for-each.js";
+import "core-js/modules/es.typed-array.from.js";
+import "core-js/modules/es.typed-array.includes.js";
+import "core-js/modules/es.typed-array.index-of.js";
+import "core-js/modules/es.typed-array.iterator.js";
+import "core-js/modules/es.typed-array.join.js";
+import "core-js/modules/es.typed-array.last-index-of.js";
+import "core-js/modules/es.typed-array.map.js";
+import "core-js/modules/es.typed-array.of.js";
+import "core-js/modules/es.typed-array.reduce.js";
+import "core-js/modules/es.typed-array.reduce-right.js";
+import "core-js/modules/es.typed-array.reverse.js";
+import "core-js/modules/es.typed-array.set.js";
+import "core-js/modules/es.typed-array.slice.js";
+import "core-js/modules/es.typed-array.some.js";
+import "core-js/modules/es.typed-array.sort.js";
+import "core-js/modules/es.typed-array.subarray.js";
+import "core-js/modules/es.typed-array.to-locale-string.js";
+import "core-js/modules/es.typed-array.to-string.js";
+import "core-js/modules/es.unescape.js";
+import "core-js/modules/es.weak-map.js";
+import "core-js/modules/es.weak-set.js";
+import "core-js/modules/esnext.aggregate-error.js";
```

Comment:
I think that it's better not inject, for example, `esnext.aggregate-error` if `es.aggregate-error` is injected, but it's another feature request.